apps/tup-cms/src/taccsite_cms/templates/snippets/css-scrolling-fixes.html

```diff
@@ -1,14 +1,24 @@
 <style id="css-scrolling-fixes">
-/* To prevent decal SVG horz. body scroll on mobile */
-@media (max-width: 575px) {
-  .container {
-      overflow-x: clip;
-  }
+/* To prevent horz. body scroll because of header nav too wide, 1092px—1066px */
+/* WARNING: In Safari 16.2–16.3, this hides header dropdown behind content */
+/* HELP: Fix the bug, instead of just hiding the bug */
+/*
+header {
+    overflow-x: clip;
 }
+*/
 
-/* Prevent trackpad scroll to show white background beyond page height */
-/* TODO: Ask Designer if "scroll away into forever" bug has come back */
-body {
-    overscroll-behavior-y: none;
```

Comment:
**Explanation.** This comes from Core-Styles now. Designer confirmed this behavior, for all clients.